.github/workflows/prs.yml

```diff
@@ -421,6 +421,8 @@ jobs:
   compute-k8s-test-plan:
     name: Compute K8s test plan
     runs-on: ubuntu-latest
+    needs: changes
+    if: ${{ needs.changes.outputs.source == 'true' }}
```

Comment:
We also need to run these for `needs.changes.outputs.helm`, and also for any docker or packaging configuration changes.